### ISSUE TITLE
Support for Xiaomi Xiaomi Viomi Internet Refrigerator iLive (462l) 10"

### ISSUE THREAD

**Issue by Keles75** [rytilahti/python-miio#567]
It's possible to add the refrigerator Xiaomi Viomi Internet Refrigerator iLive (462l) 10"
This model : https://www.xiaomitoday.com/new-viomi-refrigerator-screen/

Thank you!

**Comment by syssi** [comment]
Do you own the device? The output of

```
pip3 install python-miio
miiocli device --ip IP --token TOKEN info
```

would be helpful.

**Comment by rezmus** [comment]
viomi.fridge.v3 - Viomi Internet refrigerator iLive (French style 462L)

**Comment by syssi** [comment]
The `viomi.fridge.v3` looks like an `miot` device:

```
'setCity', [cityCode, mycity, longitude, latitude] 
'setMode', [mode]
'setOneKeyClean', [value]
'setRCSet', [value]
'setRCSetTemp', [value]
'setRCCMode', [mode]
'setCCSet', [value]
'setSceneName', [title]
'setCCSetTemp', [value]
'setFCSetTemp', [value]
'setSmartCool', [value]
'setCoolBeverage', [_value]
'setSmartFreeze', [value]
'setSceneChoose', [params]

Support properties: ['Mode', 'RCSetTemp', 'FCSetTemp', 'RCSet', 'OneKeyClean', 'FilterLifeBase', 'FilterLife', 'Error', 'SmartCool', 'SmartFreeze', 'OutdoorTemp', 'IndoorTemp', 'StartDays', 'CoolBeverage']
May be: 'IpAddr', 'SceneAll'
```

**Comment by Keles75** [comment]
Yes, this model. I can't get token because can't connect refrigerator to mihome app, only to viomi app. When I try, system wrote "did not allowed to get bind key"
![IMG_0119](https://user-images.githubusercontent.com/22985524/67714951-fd09ad00-f9d9-11e9-9ea9-c8fcacd70b87.jpeg)
And in hidden menu I find this screen:
![IMG_0131 2](https://user-images.githubusercontent.com/22985524/67715217-70abba00-f9da-11e9-8a1e-68c743254e65.jpeg)

maybe this helpful

**Comment by rezmus** [comment]
it looks like it has similar bind process to cgllc.airmonitor.b1 (Mi Multifunction Air Monitor). you setup wifi with device touch screen, than it should get bind key from xiaomi servers and expose to you as QR code. you scan it within mi home, bind is confirmed and device conneted to account.

on 2nd screen thare are probably cloud credentials (did/key/mac). i'm not even sure if it can talk via local api. i would start with fridge connection to fake hotspot or openwrt router to get network traffic.
